### ISSUE TITLE
[ISSUE] - BillableUsageAPI download() method does not process the API response. 

### ISSUE THREAD

**Issue by costichiulan** [databricks/databricks-sdk-java#162]
**Description**
 I tried to use BillableUsageAPI download method, but billableUsage API response is lost, as download method implementation is doing a .GET request with Void.class as expected response. In ApiClient, if Void type is passed as expected response, at some point, the response is lost.

```
 private <T> T execute(Request in, Class<T> target) throws IOException {
    Response out = getResponse(in);
    if (target == Void.class) {
          return null;
    }
    return deserialize(out.getBody(), target);

}

```

**Reproduction**
new AccountClient(config).billableUsage().download()

**Expected behavior**
I was expecting that a csv file with billableUsageData to be downloaded on my host. 

**Debug Logs**
The SDK logs helpful debugging information when debug logging is enabled. Set the log level to TRACE (e.g. add `log4j.logger.com.databricks=TRACE` to the log4j.properties in your project), and include the logs here.

**Other Information**
 - OS:  macOS
 - Version: 0.0.1

**Additional context**

Also it would be useful in BillableUsage API to have another method, getBillableUsage , where the actual response from the API is parsed by the ApiClient and returned. In this way, users will not be forced to add separate implementation to parse the CSV response .

**Comment by mgyucht** [comment]
Thanks for reporting this. I'm actually working on the fix now and hoping to release it in the next SDK version sometime next week.

**Comment by mgyucht** [comment]
This is fixed in the v0.10.0 release of the SDK.
